src/net/http.rs

```diff
@@ -1447,44 +1448,48 @@ impl HttpRequestType {
                 &mut R,
             ) -> Result<HttpRequestType, net_error>,
         )] = &[
-            ("GET", &PATH_GETINFO, &HttpRequestType::parse_getinfo),
-            ("GET", &PATH_GETPOXINFO, &HttpRequestType::parse_getpoxinfo),
+            ("GET", &PATH_GET_INFO, &HttpRequestType::parse_get_info),
             (
                 "GET",
-                &PATH_GETNEIGHBORS,
-                &HttpRequestType::parse_getneighbors,
+                &PATH_GET_POX_INFO,
```

Comment:
Is this "snake case" change inherently tied to your other changes? Could this be two PR's?

Reply:
I could separate it out into another PR, but since it was just renaming, it seemed like a minor enough refactor to group with this change

Reply:
Personally, I would prefer to see conceptually separable PR's separated. For reviewers, it's easier to know whether a change fits in with the necessary purpose of the change. For readers, they have an easier to follow record of why the lines changed.

But, I'm not asking you to separate them in *this* case, but maybe for the future you can consider it.

Reply:
Ya I understand this - I'll separate out the renames to a different pr!